ansible/roles/nginx/tasks/main.yml

```
@@ -36,10 +39,15 @@
   docker_container:
     name: "{{ nginx_docker_container }}"
     image: "{{ nginx_docker_image }}:{{ nginx_version }}"
-    volumes: "{{ volumes_certs }}"
+    volumes:
+      - "{{ nginx_gcs_workdir }}:/opt/gcs"
+      - "{{ volumes_certs }}"
```

```suggestion
    volumes: "{{ volumes_certs }}"
```
And add these at line 31 
```
        "{{ nginx_virtualhosts_workdir}}:/etc/nginx/conf.d/",
        "{{ nginx_gcs_workdir }}:/opt/gcs"
```
The `volumes_certs` is a list so if you want to add more volumes you have to add it in the variable and not on the `volumes` section. The current configuration for docker is like this:
```
volumes:
- /docker/nginx/gcs:/opt/gcs
- ["/docker/nginx/certs/...cert:/etc/ssl/certs..crt.", "/docker/nginx/certs/...key:/etc/ssl/...key", ...]
```

---

Ok, I got what you mean but, that means we should change the name of the variable, cause is not storing certs only anymore.

---

yes, we should change the name of the variable

---

Done